java/src/jmri/jmrix/ieee802154/xbee/XBeeTrafficController.java

```diff
@@ -31,6 +32,10 @@ public class XBeeTrafficController extends IEEE802154TrafficController implement
 
     private XBeeDevice xbee = null;
 
+    public XBeeTrafficController() {
+        super();
+    }
```

Comment:
What's the purpose of this constructor? Is it needed if there is no parameters, no code except calling super() and no other constructors in this class?

Reply:
It's not absolutely required, but it's a bit of a hedge against someone changing the super class.  If the super class has multiple constructors, you have to specify which one you're calling.